### ISSUE TITLE
ccxtpro python watch_order_book don't support slice

### ISSUE THREAD

**Issue by zsyh** [ccxt/ccxt#7960]
ccxtpro version: 0.4.64

code:
```
exchange = ccxtpro.binance()
symbol = 'BTC/USDT'
while True:
    orderbook = await exchange.watch_order_book(symbol)
    print(exchange.iso8601(exchange.milliseconds()), symbol, orderbook['asks'][:5], orderbook['bids'][:5])
```

output:
```
Traceback (most recent call last):
  File "/home/zaizai/PycharmProjects/xx/test/test_ccxtpro.py", line 19, in <module>
    asyncio.get_event_loop().run_until_complete(main())
  File "/usr/lib/python3.6/asyncio/base_events.py", line 488, in run_until_complete
    return future.result()
  File "/home/zaizai/PycharmProjects/xx/test/test_ccxtpro.py", line 17, in main
    print(exchange.iso8601(exchange.milliseconds()), symbol, orderbook['asks'][:5], orderbook['bids'][:5])
  File "/home/zaizai/PycharmProjects/xx/ccxtpro/base/order_book_side.py", line 64, in __getitem__
    if item >= self._n:
TypeError: '>=' not supported between instances of 'slice' and 'int'
```

**Comment by kroitor** [comment]
@zsyh does it work for you this way:

```Python
# -*- coding: utf-8 -*-

from asyncio import get_event_loop
import ccxtpro
from pprint import pprint


async def main(loop):
    exchange = ccxtpro.binance({
        'enableRateLimit': True, # required https://github.com/ccxt/ccxt/wiki/Manual#rate-limit
        'asyncio_loop': loop,
    })
    symbol = 'BTC/USDT'
    print('Watching', exchange.id, symbol)
    while True:
        try:
            orderbook = await exchange.watch_order_book(symbol)
            print(exchange.iso8601(exchange.milliseconds()), symbol, orderbook['asks'][:5], orderbook['bids'][:5])
        except Exception as e:
            print(str(e))
            # raise e  # uncomment to break all loops in case of an error in any one of them
            # break  # you can also break just this one loop if it fails
    await exchange.close()


if __name__ == "__main__":
    loop = get_event_loop()
    loop.run_until_complete(main(loop))
```

?

**Comment by zsyh** [comment]
@kroitor No, the output is:

```
CCXT Pro Version: 0.4.64
Watching binance BTC/USDT
GET https://api.binance.com/api/v3/exchangeInfo
GET https://api.binance.com/api/v3/exchangeInfo
'>=' not supported between instances of 'slice' and 'int'
'>=' not supported between instances of 'slice' and 'int'
'>=' not supported between instances of 'slice' and 'int'
'>=' not supported between instances of 'slice' and 'int'
'>=' not supported between instances of 'slice' and 'int'
'>=' not supported between instances of 'slice' and 'int'
'>=' not supported between instances of 'slice' and 'int'
'>=' not supported between instances of 'slice' and 'int'
'>=' not supported between instances of 'slice' and 'int'
```

**Comment by kroitor** [comment]
@zsyh got it, the fix is on the way.

**Comment by kroitor** [comment]
This should be fixed in CCXT Pro 0.4.66+, the new build will arrive in ~20 minutes:

- https://github.com/kroitor/ccxt.pro/pull/107
